lib/avrora/encoder.ex

```diff
@@ -51,6 +51,33 @@ defmodule Avrora.Encoder do
     codec.decode(payload)
   end
 
+  @doc """
+  Decode binary Avro message with :plain format and given schema.
+
+  This is pulled out as a special function to work around :plain encoded binaries that,
+  due to their encoded data, start with the magic byte.
+
+  ## Examples
+
+      ...> payload = <<0, 232, 220, 144, 233, 11, 200, 1>>
+      ...> Avrora.Encoder.decode_plain(payload,"io.confluent.numeric_transfer")
+      {:ok, %{ "link_is_enabled" => false, "updated_at" => 1_586_632_500, "updated_by_id" => 1_00 }
+  """
+  @spec decode_plain(binary(), schema_name: String.t()) ::
+          {:ok, map() | list(map())} | {:error, term()}
+  def decode_plain(payload, schema_name: schema_name) when is_binary(payload) do
+    with {:ok, schema_name} <- Name.parse(schema_name) do
+      unless is_nil(schema_name.version) do
+        Logger.warn(
+          "decoding message with schema version is not supported, `#{schema_name.name}` used instead"
+        )
+      end
+
+      schema = %Schema{full_name: schema_name.name}
+      Codec.Plain.decode(payload, schema: schema)
```

Comment:
In `decode/2` this variable was used to avoid duplication, but here it can be inlined

```suggestion
      Codec.Plain.decode(payload, schema: %Schema{full_name: schema_name.name})
```